models/P2PClient/index.js

```diff
@@ -0,0 +1,18 @@
+/*
+State:
+star = peer-star instance
+presenter = simple-peer to Presenter
+connection = true if connected to presenter, false if not
+username = name of the user (chosen by the user)
+ */
+
+module.exports = globalConfig => ({
+  namespace: 'p2c',
+  effects: require('./effects')(globalConfig),
```

Comment:
In the existing Codebase we always require it with a constant and then assign the constant.
Probably better for consistency